pkg/tlsf/contrib/include/tlsf-malloc.h

```diff
@@ -43,6 +43,28 @@
 extern "C" {
 #endif
 
+/**
+ * @brief Struct to hold the total sizes of free and used blocks
+ * Used for @ref size_walker()
+ */
+typedef struct {
+    unsigned free;          /**< total free size */
+    unsigned used;          /**< total used size */
+} tlsf_size_container_t;
```

Comment:
You use this only internally, so this should go to either an internal header file or the c-file.

Reply:
But then, how will the shell code create the struct?